content/developer/reference/frontend/javascript_modules.rst

```diff
@@ -121,15 +123,29 @@ The file :file:`file_b` can import :file:`file_a` like this:
 .. code-block:: javascript
 
   /** @odoo-module **/
-  import {something} from `./file_a`
+  import {something} from `./file_a`;
 
 But :file:`file_c` need to use the full name:
 
 .. code-block:: javascript
 
   /** @odoo-module **/
-  import {something} from `@web/file_a`
+  import {something} from `@web/file_a`;
+
+Sometimes files under `static/src` or `static/tests` should not be transpiled
+by default. It's possible to tell the transpiler to ignore those files by
+adding the tag `ignore`.
+
+.. code-block:: javascript
+
+  /** @odoo-module ignore **/
+  odoo.define("web.notTranspiled", [], function (require) {
```

Comment:
I would not use an odoo.define as example, as we don't want people to ever write that anymore. Instead, I would simply have an iife or something